### DISCUSSION TITLE
Where to find some doc of dcm4che?

### DISCUSSION THREAD

**Discussion by ChenSino** [dcm4che/dcm4che#1373] [Q&A]
Hello veryone!I searched document for dcm4che everywhere ,but I cant find it~
anyone know where is it?As a newbie ,it is hard for me to use it~

**Comment by hczedik** [comment]
For the dcm4che library and toolkit, check the README: https://github.com/dcm4che/dcm4che/blob/master/README.md

For the DCM4CHEE Archive, check the Wiki: https://github.com/dcm4che/dcm4chee-arc-light/wiki

**Reply by ChenSino** [reply]
Thanks for the reply. I use dcm4che library and I've already seen the README files,is there  usage doc,like  how to use dcm4che api to edit dicom file ?
